scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -124,37 +124,52 @@ def write_sites_only_vcf(ac_an_af_split, sites_only_vcf_path):
 
     ht = ac_an_af_rows.rows()
     ht = ht.filter(ht.alleles[1] != "*") # remove spanning deletions
+    ht = ht.checkpoint("gs://fc-eada2674-7c2b-42a6-8db3-0246872596dc/checkpoint")
     # create a filtered sites only VCF
-    hl.export_vcf(ht, sites_only_vcf_path)
+    hl.export_vcf(ht, sites_only_vcf_path, parallel='separate_header')
 
 def add_variant_tracking_info(mt, sites_only_vcf_path):
     # only need the table of row fields and leaves this as the only field
     var_ids_path = sites_only_vcf_path.replace(r".sites-only.vcf", ".var_ids.tsv.bgz")
     t = mt.rows()
-    t.select(var_origin_id=hl.format('%s-%s-%s-%s', t.locus.contig, t.locus.position, t.alleles[0], t.alleles[1])).export(var_ids_path, parallel='header_per_shard')
+    t.select(var_origin_id=hl.format('%s-%s-%s-%s', t.locus.contig, t.locus.position, t.alleles[0], t.alleles[1])).export(var_ids_path, parallel='separate_header')
 
 def main(vds, ancestry_file_location, sites_only_vcf_path):
-    transforms = [
-        hard_filter_non_passing_sites,
-        failing_gts_to_no_call,
-        remove_too_many_alt_allele_sites
-    ]
-    transformed_vds=vds
-    for transform in transforms:
-        transformed_vds = transform(transformed_vds)
+    n_parts = vds.variant_data.n_partitions()
+    n_rounds = 5
+    parts_per_round = n_parts // n_rounds
+    for i in range(n_rounds):
+        part_range = range(i*parts_per_round, min((i+1)*parts_per_round, n_parts))
+        vds = hl.vds.VariantDataset(
```

Comment:
this is clobbering the original value of `vds`, should probably be a new variable like `vds_range` or something